app/routes/cars/utils.ts

```diff
@@ -1,16 +1,16 @@
 export const colors = [
-  "red",
-  "green",
-  "blue",
-  "purple",
-  "black",
-  "gray",
-  "pink",
-  "white",
-  "orange",
-  "tan",
-  "brown",
-  "silver",
+  "Red",
+  "Green",
+  "Blue",
+  "Purple",
+  "Black",
+  "Gray",
+  "Pink",
+  "White",
+  "Orange",
+  "Tan",
+  "Brown",
+  "Silver",
 ];
 
 export function capitalize(value: string) {
```

Comment:
Do we need this function? Doesn't look to be used anywhere unless I'm missing something

Reply:
We can remove it!